core/iwasm/include/wasm_export.h

```diff
@@ -126,6 +126,22 @@ typedef WASMFunctionInstanceCommon *wasm_function_inst_t;
 struct WASMMemoryInstance;
 typedef struct WASMMemoryInstance *wasm_memory_inst_t;
 
+
+typedef struct wasm_frame_t {
+    /*  wasm_instance_t */
+    void *instance;
+    uint32_t module_offset;
+    uint32_t func_index;
+    uint32_t func_offset;
+    const char *func_name_wp;
+
+    uint32_t *sp;
+    uint8_t *frame_ref;
+    uint32_t *lp;
+} WASMCApiFrame;
+
+typedef struct wasm_frame_t *wasm_frame_ptr_t;
```

Comment:
```suggestion
typedef struct WASMCApiFrame *wasm_frame_t;
```

Reply:
I'll remove aliasing to `wasm_frame_t` completely since WASMCApiFrame can be used, to avoid confusion with `wasm_frame_t` from wasm_c_api.h